packages/wrangler/src/config/validation.ts

```diff
@@ -2266,6 +2266,21 @@ const validateContainerAppConfig: ValidatorFn = (
 			);
 		}
 
+		if (
```

Comment:
can `rollout_step_percentage` ever be `0`? If not, this could be simplified to 
```
if (containerAppOptional.rollout_step_percentage) {
  ...
}
```